sup3r/postprocessing/file_handling.py

```diff
@@ -84,6 +87,31 @@ class OutputHandler:
     back to their original form and outputting to the correct file format.
     """
 
+    @staticmethod
+    def enforce_limits(features, data):
+        """Enforce physical limits for feature data
+
+        Parameters
+        ----------
+        features : list
+            List of features with ordering corresponding to last channel of
+            data array.
+        data : ndarray
+            Array of feature data
+
+        Returns
+        -------
+        data : ndarray
+            Array of feature data with physical limits enforced
+        """
+        for i, f in enumerate(features):
+            attrs = H5_ATTRS[Feature.get_basename(f)]
+            max = attrs.get('max', np.inf)
+            min = attrs.get('min', -np.inf)
+            data[data[..., i] < min, i] = min
+            data[data[..., i] > max, i] = max
```

Comment:
I think the `np.maximum`, `np.minimum` operators will be more efficient than creating a mask array